ds2799/model-ds2799-desc-compcop_smdl.json

```
@@ -0,0 +1,29 @@
+{
+  "Name": "ds2799_compcor",
+  "Description": "scrubbing compcor, see https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds_strategy.html",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {
+    "task": "rest",
+    "space": "MNI152NLin2009cAsym"
+  },
+  "Nodes": [
+    {
+      "Level": "run",
+      "Name": "run",
+      "GroupBy": [
+        "run"
+      ],
+      "Model": {
+        "X": [
+          "rot*",
+          "trans*",
+          "a_comp_cor_??"
+        ],
+        "Options": {
+          "HighPassFilterCutoffHz" : 1,
+          "LowPassFilterCutoffHz" : 1
```

I think this would just extract the 1Hz components of the signal, getting an amplitude-modulated 1Hz sine wave. I suspect you wanted something else?

```suggestion
          "HighPassFilterCutoffHz" : 1,
          "LowPassFilterCutoffHz" : 1
```

---

yup that was in the TODO in the top of the PR: just not sure what sensible values would be for those for resting state

---

Ah, sorry. I can't read. https://www.humanbrainmapping.org/files/TemporalProcessing_Frederick_Blaise(1).pdf

---

Extracted:

Low-pass: 0.15
High-pass: 0.01